### PR TITLE
trentm/node-cmdln#6 would like hidden aliases

### PR DESCRIPTION
**Pull request by davepacheco**
There are four test failures, but they're not new failures.  I also added a test for both aliases and hidden aliases.

### CODE REVIEW

File: examples/conan.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ Conan.prototype.do_crush = function (subcmd, opts, args, callback) {
     }
     callback();
 };
+Conan.prototype.do_crush.aliases = [ 'smash' ];
+Conan.prototype.do_crush.hiddenAliases = [ 'pulverize' ];
```
Comment (trentm) [inline examples/conan.js]: merit!

**Comment by trentm** [comment]
@davepacheco Four test failures? They all pass for me: https://gist.github.com/trentm/999cfd2004ba331c1a47

Can you show me the failures you hit?

**Comment by davepacheco** [comment]
This is on OS X, Node 0.12:
https://gist.github.com/davepacheco/2da16db023666037495c

**Comment by trentm** [comment]
k, thanks, moved the issue with test failures to https://github.com/trentm/node-cmdln/issues/8

**Comment by trentm** [comment]
published as 3.2.0

**Pull request merged**